index.html

```diff
@@ -7,6 +7,25 @@
     <link rel="stylesheet" href="style.css"/>
     <!--extended Style-->
     <link rel="stylesheet" href="CSS/login.css"/>
+
+    <!--favicon-->
+    <link rel="apple-touch-icon" sizes="57x57" href="./media/favicon/apple-icon-57x57.png">
```

Comment:
Hier reicht eingentlich doch auch ein Favicon oder? Das wird vom Browser eh runterskaliert

Reply:
Hab das von nem Generator vllt is es leichter für den Browser